app/connectors/DepartureMovementConnector.scala

```diff
@@ -26,19 +27,40 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class DepartureMovementConnector @Inject() (val appConfig: FrontendAppConfig, http: HttpClient)(implicit ec: ExecutionContext) extends Logging {
 
-  def getLRN(location: String)(implicit hc: HeaderCarrier): Future[LocalReferenceNumber] = {
+  def getLRN(location: String)(implicit hc: HeaderCarrier): Future[Option[LocalReferenceNumber]] = {
 
     val headers = hc.withExtraHeaders(("Accept", "application/vnd.hmrc.2.0+json"))
 
     val url = s"${appConfig.commonTransitConventionTradersUrl}$location"
 
-    http.GET[LocalReferenceNumber](url)(HttpReads[LocalReferenceNumber], headers, ec)
+    http.GET[Option[LocalReferenceNumber]](url)(HttpReads[Option[LocalReferenceNumber]], headers, ec) recover {
+      case exception =>
+        logger.warn("getLRN failed with exception", exception)
+        None
+    }
   }
 
-  def getMessageMetaData(departureId: String)(implicit ec: ExecutionContext, hc: HeaderCarrier): Future[DepartureMessages] = {
+  def getIE015(location: String)(implicit hc: HeaderCarrier): Future[Option[IE015Data]] = {
```

Comment:
Does this need to be an option? If we cannot retrieve an IE015 we are probably going to return a technical difficulties anyways